doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -148,38 +149,30 @@ Architecture they must be implemented as per the following specifications:
+|`ra2.ntw.003`|Multiple connection points|The networking solution deployed within the implementation must support the capability to connect at least FIVE connection points to each Pod, which are additional to the default connection point managed by the default Pod network CNI plugin.|[`e.cap.004`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.ntw.004`|Multiple connection points presentation|The networking solution deployed within the implementation must ensure that all additional non-default connection points are requested by Pods using standard Kubernetes resource scheduling mechanisms such as annotations or container resource requests and limits.|[`e.cap.013`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.ntw.005`|Multiplexer/meta-plugin|The networking solution deployed within the implementation may support the use a multiplexer/meta-plugin.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)<br><br>[`req.inf.ntw.07`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.006`|Multiplexer/meta-plugin CNI Conformance|If used, the selected multiplexer/meta-plugin must integrate with the Kubernetes control plane via CNI.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)<br><br>[`req.inf.ntw.07`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.007`|Multiplexer/meta-plugin CNI Plugins|If used, the selected multiplexer/meta-plugin must support the use of multiple CNI-conformant Network Plugins.|[`req.inf.ntw.06`](chapter02.md#23-kubernetes-architecture-requirements)|
+|`ra2.ntw.008`|SR-IOV Device Plugin for Network Intensive|When hosting workloads matching the Network Intensive profile, a Device Plugin for SR-IOV may be used to configure the SR-IOV devices and advertise them to the `kubelet`.|[`e.cap.013`](chapter02.md#221-cloud-infrastructure-software-profile-capabilities)|
+|`ra2.ntw.009`|Multiple connection points with multiplexer/meta-plugin|When a multiplexer/meta-plugin is used, the additional non-default connection points must be managed by a CNI-conformant Network Plugin.|[`req.inf.ntw.08`](chapter02.md#23-kubernetes-architecture-requirements)<br><br>[`infra.net.cfg.004`](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
+|`ra2.ntw.010`|User plane networking|When hosting workloads matching the Network Intensive profile, a CNI network plugin that supports the use of DPDK, VPP, or SR-IOV must be deployed as part of the networking solution.|[`infra.net.acc.cfg.001`](chapter02.md#223-cloud-infrastructure-software-profile-requirements)|
+|`ra2.ntw.011`|NATless connectivity|When hosting workloads that require source and destination IP addresses to be preserved in the traffic headers, a CNI plugin that exposes the pod IP directly to the external networks (e.g. Calico, MACVLAN or IPVLAN) is required.|[`req.inf.ntw.14`](chapter02.md#23-kubernetes-architecture-requirements)|
```

Comment:
Calico is a solution whereas MCVLAN and IPVLAN are technology choices

Reply:
I'd argue they are well known enough as reference CNI plugins to warrant inclusion (e.g. https://www.cni.dev/plugins/main/macvlan/).

Reply:
My issue is with listing Calico

Reply:
Why? They are all open source CNI plugins that can do what the description says.

Reply:
How about `(e.g. Calico, MACVLAN or IPVLAN CNI plugins)`?

Reply:
@CsatariGergely Yes that works. Thanks

Reply:
OK- updated in https://github.com/cntt-n/CNTT/pull/2121/commits/1afa44b08815326f666da831151e0d7c9e46f957.